diff --git a/node_modules/next-auth/lib/types.d.ts b/node_modules/next-auth/lib/types.d.ts
index 816e11e..a7418b2 100644
--- a/node_modules/next-auth/lib/types.d.ts
+++ b/node_modules/next-auth/lib/types.d.ts
@@ -1,10 +1,11 @@
 import type { NextRequest } from "next/server";
+import { Awaitable } from "next-auth";
 /**
  * AppRouteHandlerFnContext is the context that is passed to the handler as the
  * second argument.
  */
 export type AppRouteHandlerFnContext = {
-    params?: Record<string, string | string[]>;
+    params: Awaitable<Record<string, string | string[]>>;
 };
 /**
  * Handler function for app routes. If a non-Response value is returned, an error
diff --git a/node_modules/next-auth/src/lib/types.ts b/node_modules/next-auth/src/lib/types.ts
index 5617581..fcc6eb0 100644
--- a/node_modules/next-auth/src/lib/types.ts
+++ b/node_modules/next-auth/src/lib/types.ts
@@ -1,4 +1,5 @@
 // @ts-expect-error Next.js does not yet correctly use the `package.json#exports` field
+import { Awaitable } from "@auth/core/types"
 import type { NextRequest } from "next/server"
 
 /**
@@ -6,7 +7,7 @@ import type { NextRequest } from "next/server"
  * second argument.
  */
 export type AppRouteHandlerFnContext = {
-  params?: Record<string, string | string[]>
+  params?: Record<string, string | string[]>;
 }
 /**
  * Handler function for app routes. If a non-Response value is returned, an error
